segmentation/include/pcl/segmentation/impl/organized_multi_plane_segmentation.hpp

```diff
@@ -91,6 +91,20 @@ pcl::OrganizedMultiPlaneSegmentation<PointT, PointNT, PointLT>::segment (std::ve
   if (!initCompute ())
     return;
 
+  // Check that the input cloud is present
+  if (!input_)
```

Comment:
`input_` check is done in `PCLBase` where this same condition exits silently via L92. So, your L94-100 are never executed

You can also choose to exit silently on L91 using `if (!initCompute() || !normals_) { return; }`

Reply:
Do we want to exit silently? This seems unhelpful to the user (me)

Reply:
> Do we want to exit silently?

No idea 😆 Just reporting what's already in the code.

Reply:
How about I remove the check on `input_` , but keep the check here on `normals_` with the error reporting? It's more in line with the other error checking in this file that way, which I feel is helpful.

Maybe we can open a separate issue to discuss if PCLBase::initCompute should report additional error messages instead of exiting silently?

Reply:
dev